eng/Version.Details.xml

```diff
@@ -6,14 +6,20 @@
     <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="6.0.0-beta.21105.5">
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>fc83e59329203724d4a63c4f6c843be62983a35e</Sha>
+      <SourceBuild RepoName="arcade" ManagedOnly="true" />
     </Dependency>
     <Dependency Name="Microsoft.DotNet.ApiCompat" Version="6.0.0-beta.21105.5">
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>fc83e59329203724d4a63c4f6c843be62983a35e</Sha>
+      <SourceBuild RepoName="arcade" ManagedOnly="true" />
```

Comment:
One of these should be removed--this just adds the same package reference twice (Microsoft.SourceBuild.Intermediate.arcade/6.0.0-beta.21105.5), but it could get out of sync somehow and cause problems. The Arcade SDK dependency seems more representative to me and the right one to keep.

(And I suppose the ArPow process should actually enforce this, eh? 😛 Filed https://github.com/dotnet/source-build/issues/2050 to do that, with more details on the problem.)

Reply:
@omajid could you address this?

Reply:
I commented this out and left a note about why. Does it look okay?

Reply:
I think it would be best to just leave it out entirely--I expect it'll be normal to have a bunch of dependencies on the same repo but only one with a `SourceBuild` element. Check out https://github.com/dotnet/aspnetcore/blob/main/eng/Version.Details.xml for example--for dotnet/runtime, we'll only mark `Microsoft.NETCore.App.Runtime.win-x64` as `SourceBuild` because it is the only package ID that is guaranteed to auto-update to the latest version even in servicing builds.

Reply:
Fixed.